api/src/main/java/org/pmiops/workbench/utils/BillingUtils.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.utils;
+
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.model.BillingAccountType;
+
+public class BillingUtils {
+  private BillingUtils() {}
+
+  public static final String BILLING_ACCOUNT_PREFIX = "billingAccounts";
+
+  public static String fullBillingAccountName(String billingAccount) {
+    return BILLING_ACCOUNT_PREFIX + "/" + billingAccount;
+  }
+
+  /**
+   * Returns {@code true} if the given billing account is workbench initial credits billing account.
+   */
+  public static boolean isInitialCredits(
```

Comment:
This class was previously named `WorkspaceUtils` and this method was named `isFreeTier()`